src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/RoboticsDataMapper.java

```diff
@@ -215,9 +230,40 @@ private Solicitor buildApplicantSolicitor(CaseData caseData, String id) {
                 caseData.getRespondent1OrganisationPolicy(),
                 "Defendant",
```

Comment:
Minor: Can we make the "Defendant" string as a final variable instead of hardcoding here. Same like the ID's. I see the defendant is used in couple of places